system/common.xml

```diff
@@ -151,6 +158,47 @@
 		</if>
 		<checkGitRepoSha repoDir="${SYSTEMTEST_ROOT}/openj9-systemtest" />
 	</target>
+	
+	<target name="clone_svttestrepo">
+		<echo message="Converting SVTTestRepo to IBM-1047 format"/>
+		<if>
+			<isset property="isZOS" />
+			<then>
+				<echo message="cleaning SVTTestRepo directories"/>
+				<delete dir="${SYSTEMTEST_ROOT}/system/SVT"/>
+				<delete dir="${SYSTEMTEST_ROOT}/system/lib"/>
+				<delete dir="${SYSTEMTEST_ROOT}/system/tools"/>
+			<exec executable="git" failonerror="false" dir="${SYSTEMTEST_ROOT}">
+				<arg value="clone" />
+				<arg value="--depth" />
+				<arg value="1" />
+				<arg value="-b"/>
+				<arg value="${svttest_repo_branch}"/>
+				<arg value="${svttest_repo}"/>
+			</exec>
+			<delete includeemptydirs="true" failonerror="false">
+				<fileset dir="${SYSTEMTEST_ROOT}/SVTTestRepo/.git/info" includes="**/*"/>
+			</delete>  
+			<mkdir dir="${SYSTEMTEST_ROOT}/SVTTestRepo/.git/info" />
+			<move file="${SYSTEMTEST_ROOT}/SVTTestRepo/.gitattributes.zos" tofile="${SYSTEMTEST_ROOT}/SVTTestRepo/.git/info/attributes" /> 
+			<exec executable="git" dir="${SYSTEMTEST_ROOT}/SVTTestRepo" failonerror="false">
+				<arg value="rm" />
+				<arg value="--cached" />
+				<arg value="-r" />
+				<arg value="-q" />
+				<arg value="." />
+			</exec>
+			<exec executable="git" dir="${SYSTEMTEST_ROOT}/SVTTestRepo" failonerror="false">
```

Comment:
I see that you have basically done here what we did for the [JCK compilation story on z/OS](https://github.com/adoptium/aqa-tests/blob/master/jck/build.xml#L138).

The question is why? Do you see some of the SVT tests failing on z/OS due to conversion issues? Please note that we do not need to do these extra steps (remove and hard reset) on aqa system tests. 

Also, how long of a delay does this extra rm and reset steps introduce on the SVT repo compilation?

Reply:
Yes, SVT tests building is failing on z/OS as files were not converted. Hence I'm trying to convert SVTTestRepo on z/OS.

Reply:
Actually I tried to understand how openjdk-systemtest is getting converted and tried the same way for SVTTestRepo.

Reply:
We're running SVT tests on z/OS on Jenkins for the first time, so we are not sure what is the regular conversion. Could you please point us where regular conversion happens.

Reply:
By regular conversion, I referred to the conversion performed by Git itself (https://git-scm.com/docs/gitattributes/2.19.2).